src/app/resources/patreon.py

```diff
@@ -38,12 +38,18 @@ def get(self):
         else:
             return redirect(f"{cfg.API['website-url']}/linkpatreon?error=unknown")
 
-        return redirect(f"{cfg.API['website-url']}/linkpatreon?error=unknown")
 
+class PatreonLinkSchema(Schema):
+    ckey = fields.String()
+    patreon_id = fields.String()
 
-class LinkedPatreonListResource(Resource):
-    def get(self):
-        if request.args.get("pass") == cfg.PRIVATE["api_passwd"]:
+
+class LinkedPatreonListResource(MethodResource):
+    @marshal_with(PatreonLinkSchema(many=True))
+    @use_kwargs(APIPasswordRequiredSchema)
+    @doc(description="Get a list of linked ckey-Patreon accounts.")
+    def get(self, **kwargs):
+        if kwargs["api_pass"] == cfg.PRIVATE["api_passwd"]:
```

Comment:
renaming the arg from `pass` to `api_pass` seems wrong, since the `APIPasswordRequiredSchema` only lists `pass` as the parameter

Reply:
pass is a keyword in python so I couldn't define it directly in the schema, but I didn't want to deprecate the current implementation of the endpoint, so I changed the data label in the schema to match. I believe I did test this and saw that it worked as is.